client.go

```diff
@@ -353,11 +353,14 @@ func (s *Client) LoopOutQuote(ctx context.Context,
 		return nil, ErrSwapAmountTooHigh
 	}
 
-	logger.Infof("Offchain swap destination: %x", terms.SwapPaymentDest)
+	quote, err := s.Server.GetLoopOutQuote(ctx, request.Amount)
+	if err != nil {
+		return nil, err
+	}
 
-	swapFee := swap.CalcFee(
-		request.Amount, terms.SwapFeeBase, terms.SwapFeeRate,
-	)
+	logger.Infof("Offchain swap destination: %x", quote.SwapPaymentDest)
```

Comment:
Why add this?

Reply:
It was there from before, just on the terms instead of quote.